test/README.md

```diff
@@ -4,7 +4,8 @@ lorax uses Cockpit's integration test framework and infrastructure. To do this,
 we're checking out Cockpit's `bots/` subdirectory. It contains links to test
 images and tools to manipulate and start virtual machines from them.
 
-Each test is run on a new instance of a virtual machine. 
+Each test is run on a new instance of a virtual machine.
+Branch/test matrix is configured in `bots/tests-scan`.
```

Comment:
Please mention that it's configured on the [cockpit-project/cockpit](https://github.com/cockpit-project/cockpit) repository.